server/app/views/applicant/NavigationFragment.html

```diff
@@ -36,6 +37,96 @@
   </header>
 </th:block>
 
+<section
+  th:fragment="officialGovermentWebsiteBanner"
+  class="usa-banner"
+  aria-label="Official website of the United States government"
+>
+  <div class="usa-accordion">
+    <header class="usa-banner__header">
+      <div class="usa-banner__inner">
+        <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
+          <p
+            class="usa-banner__header-text"
+            th:text="#{banner.northStarTitle}"
+          ></p>
+          <p class="usa-banner__header-action" th:text="#{banner.link}"></p>
+        </div>
+        <button
+          type="button"
+          class="usa-accordion__button usa-banner__button"
+          aria-expanded="false"
+          aria-controls="gov-banner-default"
+        >
+          <span class="usa-banner__button-text" th:text="#{banner.link}"></span>
+        </button>
+      </div>
+    </header>
+    <div
+      class="usa-banner__content usa-accordion__content"
+      id="gov-banner-default"
+    >
+      <div class="grid-row grid-gap-lg">
+        <div class="usa-banner__guidance tablet:grid-col-6">
+          <img
+            class="usa-banner__icon usa-media-block__img"
+            th:src="${govIcon}"
+            role="img"
+            alt=""
+            aria-hidden="true"
+          />
+          <div class="usa-media-block__body">
+            <p>
+              <strong th:text="#{banner.govWebsiteSectionHeader}"></strong
+              ><br />
+              <span th:text="#{banner.govWebsiteSectionContent}"></span>
+            </p>
+          </div>
+        </div>
+        <div class="usa-banner__guidance tablet:grid-col-6">
+          <img
+            class="usa-banner__icon usa-media-block__img"
+            th:src="${httpsIcon}"
+            role="img"
+            alt=""
+            aria-hidden="true"
+          />
+          <div class="usa-media-block__body">
+            <p>
+              <strong th:text="#{banner.httpsSectionHeader}"></strong><br />
+              <th:block
+                th:utext="#{banner.httpsSectionContent(~{this :: lockIcon})}"
+              ></th:block>
+            </p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<span th:fragment="lockIcon" class="icon-lock"
```

Comment:
**suggestion:** I'm not sure exactly where its defined, but can we use the icon fragment that we use in some other places? https://github.com/civiform/civiform/blob/87629521179c78a9f29f6a42b43479327e569e75/server/app/views/applicant/ApplicantBaseFragment.html#L68

Reply:
so annoyingly we can't use the Icon fragment because we're passing html into the httpsSectionContent message, meaning we have to use th:utext, but then it means it won't process anything within the inner template..

I added some content to the wiki about this, but one of my biggest annoyances with thymeleaf is adding HTML within a message. It's possible but you can't have any variables within the HTML... We're also adding a couple custom classes so it appears inline with the banner.

I added a comment to the fragment here though

Reply:
Thanks for clarifying!